core/nodepool/config/templates/stack-template.json

```diff
@@ -266,6 +266,9 @@
         {{else}}
         "PlacementTenancy": "{{.Tenancy}}",
         {{end}}
+        {{if EbsOptimized .InstanceType }}
```

Comment:
Just wanted to note that I slightly prefer `.InstanceType.EbsOptimized`, while turning `.InstanceType` to its own type and writing tests for it in go. But that's not a strong opinion.

Reply:
Please take it as I didn't want to make things too complicated and hence my comment at https://github.com/kubernetes-incubator/kube-aws/issues/1252#issuecomment-385591265 😄